tests/sentry/tasks/test_backfill_seer_grouping_records.py

```diff
@@ -1776,13 +1800,20 @@ def test_backfill_seer_grouping_records_cohort_creation_multiple_batches(
         """
         Test that when the cohort size is 1, multiple cohorts of size 1 are created and processed.
         """
-        # Create 2 seer eligible projects that project_id % thread_number == worker_number
-        thread_number = options.get("similarity.backfill_total_worker_count")
-        worker_number = self.project.id % thread_number
+        total_workers = options.get("similarity.backfill_total_worker_count")
+        worker_number = self.get_worker_number_for_project(self.project, total_workers)
+
+        # Create a project whose worker number matches that of `self.project`, to prove they
+        # nonetheless end up in different cohorts. Regenerate the project if necessary until the
```

Comment:
I think we can do something like `create_project(id=self.project + 1)` to ensure they're in different cohorts

Reply:
No, I don't think we can count on that. The hashing that postgres does is kind of a black box, so it's entirely possible for consecutive project ids to both hash to values with the same mod value.